scripts/run-local.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+server_local_development_env () {
```

Comment:
Some of us have multiple tenants and are switching between them. So instead of having a script that uses what ever I'm logged in to in a given moment, let's simply rely on configuration file.

Reply:
updated